documentation/modules/managing/con-consumer-config-properties.adoc

```diff
@@ -65,6 +79,30 @@ group.id=my-group-id <1>
 ----
 <1> Add a consumer to a consumer group using a group id.
 
+== Choosing the right partition assignment strategy
+
+Select an appropriate partition assignment strategy, which determines how Kafka topic partitions are distributed among consumer instances in a group.
```

Comment:
@showuon Should we recommend users to set the strategy to `CooperativeStickyAssignor`? I remember a few issues with it, have they been all solved? 
By default consumers still use `RangeAssignor` which in many cases is literally the worst option.

Reply:
Yes, AFAIK, there's a bug related to CooperativeStickyAssignor that could cause duplicated consumption still open: https://issues.apache.org/jira/browse/KAFKA-14224 . But yes, rangeAssignor is not a good default one, but I think that's a safe one. Here, I think it's better we don't recommend any assignor and let users to choose it.

Reply:
Thanks. We don't recommend a strategy and also suggest that a custom strategy might work better, so we can leave the section as it is.